Penumbra/UI/ModsTab/ModPanelEditTab.cs

```diff
@@ -80,11 +83,34 @@ public void DrawContent()
             }
 
         UiHelpers.DefaultLineSpace();
+        var sharedTagsEnabled = _sharedTagManager.SharedTags.Count() > 0;
+        var sharedTagButtonOffset = sharedTagsEnabled ? ImGui.GetFrameHeight() + ImGui.GetStyle().FramePadding.X : 0;
         var tagIdx = _modTags.Draw("Mod Tags: ", "Edit tags by clicking them, or add new tags. Empty tags are removed.", _mod.ModTags,
-            out var editedTag);
+            out var editedTag, rightEndOffset: sharedTagButtonOffset);
         if (tagIdx >= 0)
             _modManager.DataEditor.ChangeModTag(_mod, tagIdx, editedTag);
 
+        if (sharedTagsEnabled)

```

Comment:
Is this exactly the same as before? Then that separate function should probably be accessible for both :D

Reply:
Yeah these are identical except for the last bool in the `_sharedTagManager.DrawAddFromSharedTags()` call being flipped. I initially had them here from near when I started working on this and the logic was still pretty simple, and then never split them out once it got more complicated.

I'll split this out into a separate function (probably within SharedTagManager).